taverna-activity-palette-impl/pom.xml

```diff
@@ -92,7 +94,7 @@
 			<groupId>org.apache.taverna.osgi</groupId>
 			<artifactId>taverna-app-configuration-impl</artifactId>
 			<version>${taverna.osgi.version}</version>
-			<scope>test</scope>
```

Comment:
Could you re-add this scope? I don't see that it needs a runtime dependency here, it compiles well with `<scope>test</scope>`